modules/KalturaSupport/components/logo.js

```diff
@@ -9,9 +9,9 @@
 			displayImportance: 'low',
 			align: "right",
 			cssClass: "kaltura-logo",
-			href: null,
-			title: null,
-			img: null
+			href: 'http://www.kaltura.com',
+			title: 'Kaltura',
+			img: 'http://kgit.html5video.org/branches/master//skins/kdark/images/kalturaLogo.png'
```

Comment:
@MaksymKorniiov  
We cannot hard-code a release with path to kgit. Let's try to change this to a relative path and if not possible I will work with you on an absolute kaltura CDN file.